### ISSUE TITLE
IllegalArgumentException when requesting run-time permissions

### ISSUE THREAD

**Issue by ridsatrio** [ridsatrio/kopi#4]
Requesting permission using the helper class `Permissions` gave out an `IllegalArgumentException`. The stack trace stated that we should, "...only use lower 8 bits for requestCode".

Presumably, this is caused by the usage of bit-flag-styled numbering for request codes used in said class.

Here's the full stack trace:

```
Caused by: java.lang.IllegalArgumentException: Can only use lower 8 bits for requestCode
   at android.support.v4.app.FragmentActivity.validateRequestPermissionsRequestCode(FragmentActivity.java:799)
   at android.support.v4.app.ActivityCompatApi23.requestPermissions(ActivityCompat23.java:29)
   at android.support.v4.app.ActivityCompat.requestPermissions(ActivityCompat.java:316)
   at id.ridsatrio.kopi.security.Permissions.request(Permissions.java:155)
   at id.ridsatrio.kopi.security.Permissions.requestCamera(Permissions.java:54)
   at id.ridsatrio.kopi.sample.DemoActivity.onCreate(DemoActivity.java:14)
   at android.app.Activity.performCreate(Activity.java:6237)
   at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
   at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
   at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
   at android.app.ActivityThread.-wrap11(ActivityThread.java) 
   at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
   at android.os.Handler.dispatchMessage(Handler.java:102) 
   at android.os.Looper.loop(Looper.java:148) 
   at android.app.ActivityThread.main(ActivityThread.java:5417) 
   at java.lang.reflect.Method.invoke(Native Method) 
   at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
   at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

**Comment by ridsatrio** [comment]
Solved alongside commit 6769555.

**Comment by ridsatrio** [comment]
Solved alongside commit 6769555.

**Comment by ridsatrio** [comment]
Issue reopened. I made a mistake in the working branch and have decided to delete said branch and re-do things over.

**Comment by ridsatrio** [comment]
Solved (again) alongside commit 1546961.
